about.html

```diff
@@ -12,57 +12,73 @@
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
+  <div class="container">
+    <section>
+      <nav>
+          <img src='/img/lambda-black.png'>
+          <a href="index.html"><p>Home</p></a>
+          <a href="about.html"><p>About</p></a>
+          <a href="products"><p>Products</p></a>
+          <a href="blog"><p>Blog</p></a>
+          <a href="contact"><p>Contact</p></a>
+      </nav>
+      <img src='img/jumbo-about.png' id='heroImage'>
+    </section>
+    <section class="middle-content">
+      <h2>About</h2>
+
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+    </section>
+    <div class='learnBoxes'>
+      <section class="middle-content">
+        <img src="img/about-plan.png" alt="strategy">
+        
+        <h2>Strategy</h2>
+        
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+        <button>Learn More</button>
+      </section>
 
-    Learn More
+      <section class="middle-content">
+        <img src="img/about-working.png" alt="strategy">
+        
+        <h2>How We Work</h2>
+        
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+
+        <button>Learn More</button>
+      </section>
+    </div>
+    <div class='learnBoxes'>
+      <section class="middle-content">
+        <img src="img/about-office.png" alt="strategy">
+        
+        <h2>Places We Work</h2>
 
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Learn More
+        <button>Learn More</button>
+      </section>
+      <section class="middle-content">
+        <img src="img/about-meeting.png" alt="strategy">
```

Comment:
You have the right idea but maybe forgot about it, the alt attribute in each img tag is set to "strategy". For accessibility purposes, it's recommended that the attribute reflect the sections more accurately.